app/Models/Category.php

```diff
@@ -98,14 +98,14 @@ public function _feeds($values) {
 	}
 
 	public function _attributes($key, $value) {
-		if ($key == '') {
+		if ('' === $key) {
```

Comment:
Probably `==` is better there. It would still be a problem with null and 0 and false